Dockerfile

```diff
@@ -1,4 +1,8 @@
-FROM alpine:3.14
+ARG GITPOD_VERSION="aledbf-insts3.1"
```

Comment:
Do we need to document the path to get these on the standard versions?

Reply:
fixed